hw/formal/tools/dvsim/common_conn_cfg.hjson

```diff
@@ -2,22 +2,22 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 {
-  flow:             fpv
+  flow:             formal
   sub_flow:         fconn
```

Comment:
s/fconn/conn?

To me, `conn` seems sufficient (fconn is not conveying anything extra - the file path already has 'formal' in it so I know it refers to formal-based connectivity testing). Would actually prefer 'connectivity' but 'conn' is fine too.